Index: src/Notes
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Notes	(revision 978dd7bf100aa49df6ebf7a0523f10dc730dbf8f)
+++ src/Notes	(revision )
@@ -6,4 +6,21 @@
 >Char
 >Boolean
 
-Objects
\ No newline at end of file
+Objects
+
+
+-----------
+Binary
+base 2
+
+Byte 8 bits
+
+base 8= 256
+↕︎
+unique valeus 255
+
+-----------
+# of unique values=256
+largest number = 255 in 2/\8
+
+-----------
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/vcs.xml	(revision )
+++ .idea/vcs.xml	(revision )
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
